pkg/app/server/grpcapi/grpcapi.go

```diff
@@ -184,10 +184,6 @@ func gRPCStoreError(err error, msg string) error {
 	return status.Error(codes.Internal, fmt.Sprintf("Failed to %s", msg))
 }
 
-func makeUnregisteredAppsCacheKey(projectID string) string {
-	return fmt.Sprintf("HASHKEY:UNREGISTERED_APPS:%s", projectID)
-}
-
```

Comment:
Added: #2847
Unrefered: #3326

[This](https://github.com/kentakozuka/pipecd/blob/4692fbbe5c848c4e54b6399e8584829e35010ab0/pkg/app/server/unregisteredappstore/store.go#L102-L104) is currently used instead.